pareto/examples/multiobjective_optimization/seismicity_vs_cost_MOO.ipynb

```diff
@@ -0,0 +1,531 @@
+    "    \"pareto.case_studies\",\n",
+    "    \"strategic_toy_case_study.xlsx\",\n",
+    ") as fpath:\n",
+    "    [df_sets, df_parameters] = get_data(fpath, set_list, parameter_list)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "89073813-543a-4664-83c0-8db01339c46e",
+   "metadata": {},
+   "source": [
+    "## Epsilon constraint method\n",
+    "\n",
+    "The first MOO algorithm we will implement is known as the epsilon constraint method. We begin by creating the PARETO model:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4cf676f9-3c21-4c0d-8a43-4fb2e6750ea3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Model settings\n",
+    "default = {\n",
+    "    \"objective\": Objectives.cost,\n",
+    "    \"pipeline_cost\": PipelineCost.distance_based,\n",
+    "    \"pipeline_capacity\": PipelineCapacity.input,\n",
+    "    \"hydraulics\": Hydraulics.false,\n",
+    "    \"node_capacity\": True,\n",
+    "    \"water_quality\": WaterQuality.false,\n",
+    "    \"removal_efficiency_method\": RemovalEfficiencyMethod.concentration_based,\n",
+    "    \"infrastructure_timing\": InfrastructureTiming.false,\n",
+    "    \"subsurface_risk\": SubsurfaceRisk.exclude_risky_wells,\n",
+    "}\n",
+    "\n",
+    "# Create model\n",
+    "model = create_model(\n",
+    "    df_sets,\n",
+    "    df_parameters,\n",
+    "    default=default,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e74dc7af-2340-47c1-a7a6-d802e1531eec",
+   "metadata": {},
+   "source": [
+    "Next, we set the solver options, solve the model, and check that the solution is feasible and optimal."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f42b8b21-8f1e-4b22-97e3-e0f8a4cc1ae9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "options = {\n",
+    "    \"deactivate_slacks\": True,\n",
+    "    \"scale_model\": False,\n",
+    "    \"scaling_factor\": 1000,\n",
+    "    \"running_time\": 200,\n",
+    "    \"gap\": 0,\n",
+    "}\n",
+    "\n",
+    "results_min_cost = solve_model(model=model, options=options)\n",
+    "\n",
+    "assert results_min_cost.solver.termination_condition == TerminationCondition.optimal\n",
+    "assert results_min_cost.solver.status == SolverStatus.ok\n",
+    "\n",
+    "with nostdout():\n",
+    "    feasibility_status = is_feasible(model)\n",
+    "\n",
+    "assert feasibility_status"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ed091ed2-f38c-4025-b9bc-6b969acb26e2",
+   "metadata": {},
+   "source": [
+    "We investigate the value of the objective function (minimum cost) and the corresponding subsurface risk:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c6e5b284-d42e-4346-b353-11a4115adb00",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "min_cost_obj = model.v_Z.value\n",
+    "min_cost_risk = model.v_Z_SubsurfaceRisk.value\n",
+    "\n",
```

Comment:
With the changes of Pyomofication of the subsurface_risk_metric, one might have to change the min_cost_risk to be equal to `model.v_Z_SubsurfaceRisk.value + model.subsurface.y_dist.value` as there is a change in the objective function for the logic to work.

Reply:
@JavalVyas2000 I changed the subsurface risk objective function so that your suggestion is not necessary.